python/paddle/fluid/trainer.py

```diff
@@ -431,6 +431,28 @@ def save_params(self, param_path):
             exe = executor.Executor(self.place)
             io.save_persistables(exe, dirname=param_path)
 
+    def save_inference_model(self, param_path, feeded_var_names,
+                             target_var_indexes):
+        """
+        Save model for cpp inference into :code:`param_path`.
+
+        Args:
+            param_path(str): The path to save parameters.
+            feeded_var_names(list(str)): The name of the vars that you
+                need to feed in before run program.
+            target_var_indexes(list(int)): the index of target var that
+                you need to return in trainer.train_func.
```

Comment:
not sure if user know how to use it...I'm hoping that Trainer/Inferencer are in contrib namespace

Reply:
I agree. Currently, trainer is lack of maintenance.